install/kubernetes/cilium/templates/cilium-operator/poddisruptionbudget.yaml

```diff
@@ -12,8 +12,10 @@ metadata:
     app.kubernetes.io/part-of: cilium
 spec:
   {{- with $component.maxUnavailable }}
+  {{- if ne . "" }}
```

Comment:
From [Helm's `with` documentation](https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with):

> […] the block after `with` only executes if the value of `PIPELINE` is not empty.

If I understand correctly then the condition `ne . ""` will always evaluate to `true`.